test/mocks/JobForTest.sol

```diff
@@ -2,9 +2,13 @@
 pragma solidity 0.8.19;
 
 import {Job, IJob} from '@contracts/jobs/Job.sol';
+import {Authorizable} from '@contracts/utils/Authorizable.sol';
 
 contract JobForTest is Job {
-  constructor(address _stabilityFeeTreasury, uint256 _rewardAmount) Job(_stabilityFeeTreasury, _rewardAmount) {}
+  constructor(
+    address _stabilityFeeTreasury,
+    uint256 _rewardAmount
+  ) Job(_stabilityFeeTreasury, _rewardAmount) Authorizable(msg.sender) validParams {}
```

Comment:
`Job` does not `validParams` in its scope because then inheriting contracts that try to do the same revert during setup.

Reply:
wdym? but the ultimate constructor will validate params no?

Reply:
Yes, that's what it should happen, but it's not enforced. 🧐